conda_smithy/update_cb3.py

```diff
@@ -215,19 +220,20 @@ def update_cb3(recipe_path, conda_build_config_path):
                     need_mingw_c = True
                 continue
             if req_rendered == 'cython' and not (need_c or need_cxx or need_f):
-                messages['Found cython requirement. Adding compiler'] = True
-                need_c = True
+                if 'c' not in compilers:
+                    messages['Found cython requirement. Adding compiler'] = True
+                    need_c = True
             if req in ['ninja', 'jom', 'cmake', 'automake', 'autoconf', 'libtool',
                        'make', 'pkg-config', 'automake-wrapper', 'posix', 'm4'] \
                     or req.startswith("{{p") or req.startswith("m2-") \
                     or (req_rendered in ['perl', 'texlive-core', 'curl', 'openssl', 'tar', 'gzip', 'patch']
-                        and section == 'build' and req_rendered not in reqs['run']):
+                        and req_rendered not in reqs['run']) and section == 'build':
```

Comment:
These should be in new `build` section. If already done, they'll be in `build` section, no need to move.

Reply:
I think I meant `section == 'host'`